Final_Project_Paper-AR.qmd

```diff
@@ -0,0 +1,174 @@
+  add_row(word="7", lexicon="NA")%>%
+  add_row(word="ing", lexicon="NA")%>%
+  add_row(word="de", lexicon="NA")%>%
+  add_row(word="tion", lexicon="NA")%>%
+  add_row(word="vera", lexicon="NA")%>%
+  add_row(word="vou", lexicon="NA")%>%
+  add_row(word="editor", lexicon="NA")%>%
+  add_row(word="subscription", lexicon="NA")%>%
+  add_row(word="page", lexicon="NA")%>%
+  add_row(word="printed", lexicon="NA")%>%
+  add_row(word="ment", lexicon="NA")%>%
+  add_row(word="matter", lexicon="NA")%>%
+  add_row(word="publications", lexicon="NA")%>%
+  add_row(word="con", lexicon="NA")%>%
+  add_row(word="street", lexicon="NA")
+
+tidy_fp <- fp_whole %>%
+  unnest_tokens(word, text) %>% 
+  filter(str_detect(word, "[a-z']$")) %>% 
+  anti_join(stop_words_custom)
+
+tidy_fp <- tidy_fp %>% filter(!grepl('[0-9]', word))
+```
+
+The Foreign Policy Bulletin Dataset provides insights on historical foreign policy events considered significant and relevant by individuals engaging with the U.S. foreign relations community. Digital methods, particularly text analysis, reveals paradigms related to the relevance of the Latin American nations regarding U.S. foreign policy efforts during the interwar period and World War II. From counting words to using topic modeling and word vectors, text analysis has illuminated important insights on the significance of Latin America to the United States, as well as the themes and words associated most with Latin American nations. When examining the direct word references, it becomes apparent that Latin American nations were discussed far less than European nations and the Axis Powers.
+
+```{r}
+#country word count
+tidy_fp_countries <- tidy_fp %>% 
+  filter(word %in% c("brazil", "argentina", "germany", "britain", "japan", "mexico", "soviet", "russia", "america", "chile", "venezuela")) %>% 
+  count(word, sort = TRUE)
+ 
+
+tidy_fp_countries
+```
+In the Foreign Policy Bulletin Dataset, European nations and the Axis Powers are disproportionately focused on. In raw numbers, from 1936-1946, Britain, Germany, Russia, and Japan were directly mentioned 12,027 times. Meanwhile, Argentina, Brazil, Chile, Mexico, and Venezuela were only directly referenced 1,357 times. Yet, the bulletin’s lackluster coverage of Latin America in juxtaposition to its European counterparts should not diminish Latin America’s significance to U.S. foreign policy. The major theme associated with Latin American nations in the bulletin indicates the critical role that Latin America played in protecting U.S. interests. 
+
+```{r}
+#topic model
+tidy_fp_words <- tidy_fp %>% count(filename, word)
+
+fp.dtm <- tidy_fp_words %>% 
+  count(filename, word) %>% 
+  cast_dtm(filename, word, n)
+
+fp3.lda <- LDA(fp.dtm, k = 16, control = list(seed = 12345), method = "Gibbs", alpha = 0.5)
+fp3.lda
+
+fp3.topics <- tidy(fp3.lda, matrix = "beta")
+
+fp3.top.terms <- fp3.topics %>%
+  arrange(desc(beta)) %>% 
+  group_by(topic) %>% slice(1:10)
+
+fp3.top.terms %>%
```

Comment:
def look into topics over time - I think it might yield some interesting research questions for you.